hack/migration/ds.yaml

```diff
@@ -99,13 +99,15 @@ spec:
   # You must be using a CSI volume driver. CSI drivers are now the default on GCP clusters.
   # For more information on CSI drivers, see: https://kubernetes.io/docs/concepts/storage/volumes/#csi
   snapshots:
-    enabled: true
+    enabled: false
```

Comment:
Just to confirm that this file and the similar file in the forgeops repo are being changed in tandem.

I wonder if this file even needs to exist, now that there's a version in the forgeops repo. A directory named `hack` in open source code is not the greatest naming choice.

Reply:
I will change the forgeops version afterwards. This change isn't automatically released so won't be used by default. I'll change forgeops prior to updating the shared cluster.

Reply:
I think we need to keep a copy of this file here because a customer doesn't necessarily have to use forgeops too. I agree with the naming convention.  The folder is probably better named samples or something similar. I'll create a ticket to revise.

Reply:
Just some history since I'm still on the PR ;-)  The name "hack" is canonical - it's what the Kubebuilder framework creates.